services/notifications/docs/Notifications.md

```diff
@@ -0,0 +1,214 @@
+# Notifications
+
+## GET /notifications/
```

Comment:
Since we're using the `-------` format of making titles elsewhere instead of the `#` format, let's keep that consistent here.